Parser does not compile with Antlr4.5 from homebrew

I'm on a mac. Have installed .NET using the instructions here https://github.com/aspnet/home

Used homebrew to install Antlr4. Which resolves to 4.5.

Compiling my generated files I get the following errors
error CS0534: 'MyParser' does not implement inherited abstract member 'Recognizer<IToken, ParserATNSimulator>.TokenNames.get'

MyParser.cs(437,41): error CS0103: The name 'TokenStream' does not exist in the current context

I can change TokenStream to _input and it resolves those errors, not sure about the other one.

I guess this might be an issue that the Nuget Antlr4.Runtime (how I reference this) is different to the one used in the released Antlr 4.5 version? Pretty sure .net on the mac is just Mono.

Any ideas?

Thanks

--- a/build/Antlr4.CodeGenerator.nuspec
+++ b/build/Antlr4.CodeGenerator.nuspec
@@ -46,6 +46,10 @@
 
     <file src="..\runtime\CSharp\Antlr4BuildTasks\bin\net40\$Configuration$\Antlr4.net40.props" target="build\Antlr4.CodeGenerator.props"/>
     <file src="..\runtime\CSharp\Antlr4BuildTasks\bin\net40\$Configuration$\Antlr4.net40.targets" target="build\Antlr4.CodeGenerator.targets"/>
-    <file src="..\runtime\CSharp\Antlr4BuildTasks\bin\net40\$Configuration$\Antlr4BuildTasks.dll" target="build"/>
+
+    <!-- Build Tasks -->
+
+    <file src="..\runtime\CSharp\Antlr4BuildTasks\bin\net40\$Configuration$\Antlr4BuildTasks.dll" target="tools\net40"/>
+    <file src="..\runtime\CSharp\Antlr4BuildTasks\bin\netstandard\$Configuration$\Antlr4BuildTasks.dll" target="tools\netstandard"/>
   </files>
 </package>

--- a/runtime/CSharp/Antlr4BuildTasks/Antlr4.net40.props
+++ b/runtime/CSharp/Antlr4BuildTasks/Antlr4.net40.props
@@ -1,8 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <Antlr4NetStandard Condition="'$(MSBuildToolsVersion)' == '2.0' OR
+                      '$(MSBuildToolsVersion)' == '3.5' OR
+                      '$(MSBuildToolsVersion)' == '4.0' OR
+                      '$(MSBuildToolsVersion)' == '12.0'">False</Antlr4NetStandard>
+    <Antlr4NetStandard Condition="'$(Antlr4NetStandard)' == ''">True</Antlr4NetStandard>
     <!-- Folder containing Antlr4BuildTasks.dll -->
-    <Antlr4BuildTaskPath>.</Antlr4BuildTaskPath>
+    <Antlr4BuildTaskPath Condition="'$(Antlr4NetStandard)' != 'True'">..\tools\net40</Antlr4BuildTaskPath>
+    <Antlr4BuildTaskPath Condition="'$(Antlr4NetStandard)' == 'True'">..\tools\netstandard</Antlr4BuildTaskPath>
     <!-- Path to the ANTLR tool itself -->
     <Antlr4ToolPath>..\tools\antlr4-csharp-4.6.2-SNAPSHOT-complete.jar</Antlr4ToolPath>
   </PropertyGroup>

